.github/workflows/ci_tests.yaml

```diff
@@ -45,11 +45,14 @@ jobs:
           #   python-version: 3.7
           #   isDraft: true
         # Pair Python 3.7 with NumPy 1.17 and Python 3.9 with NumPy 1.20
+        # Only install geopandas on Python 3.9/NumPy 1.20
```

Comment:
What's the reason for this?

Reply:
Since geopandas is an optional dependency, probably good to have a test that doesn't include it.